src/OpenSage.Game/Data/Scb/ScriptPlayer.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.IO;
+using OpenSage.Data.Map;
+using OpenSage.FileFormats;
+
+namespace OpenSage.Data.Scb
+{
+    public sealed class ScriptPlayer
```

Comment:
Or should this be Script*s*Player?

Reply:
I guess so, if you don’t mind renaming it, otherwise I’m happy to merge as is.